exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,12 @@
 let name = "Daniel";
 let danielsRole = "mentor";
 
+if (danielsRole === "mentor") {
+  console.log("Hi, I'm Daniel, I'm a mentor.");
```

Comment:
The logic here is good. But it might be a good idea to use the name variable here instead of repeating "Daniel" in the code. Imagine, if we wanted to change the name, we could just update a single variable instead of having to update everywhere in the code that mentions "Daniel"